addons/t9n/models/message.py

```diff
@@ -11,14 +11,67 @@ class Message(models.Model):
     _description = "Localizable message"
 
     body = fields.Text(
+        "Entry to be Translated",
         help="The actual, textual content to be translated.",
```

Comment:
This is probably cleaner English

```suggestion
        "Text to Translate"
```